assets/dev/js/editor/controls/base-data.js

```diff
@@ -216,9 +216,12 @@ ControlBaseDataView = ControlBaseView.extend( {
 		const parent = this.getResponsiveParentView();
 
 		if ( parent ) {
-			const placeholder = parent.preparePlaceholderForChildren();
+			const name = this.model.get( 'name' ),
+				placeholder = '_inline_size_mobile' === name ?
```

Comment:
It shouldn't be hardcoded. you can configure the control like

```
$this->add_responsive_control(
			'_inline_size',
			[
				'label' => __( 'Column Width', 'elementor' ) . ' (%)',
				'type' => Controls_Manager::NUMBER,
				'responsive_placeholder' => 100,
```

and here you will use this config